{{cookiecutter.project_slug}}/src/features/{% if cookiecutter.has_messenger_blueprint == "y" %}Messenger{% endif %}/screens/ChatScreen/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { GiftedChat } from 'react-native-gifted-chat'
+import PubNubReact from 'pubnub-react';
+
+export default class ChatScreen extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.pubnub = new PubNubReact({
+      publishKey: 'pub-c-09038c51-029c-47a7-95f2-b6c7407afb12',
```

Comment:
Please move this keys to the config file

https://github.com/crowdbotics/react-native-scaffold/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/src/config/app.js